htdocs/lib2/menu.class.php

```diff
@@ -34,7 +36,7 @@ public function __construct()
         // add country-dependent town list for small map
         $country = $login->getUserCountry();
         if ($opt['map']['towns']['enable'] &&
-            isset($build_map_towns_menu) && $build_map_towns_menu &&   // optimization
+            isset($build_map_towns_menu) && $build_map_towns_menu && // optimization
```

Comment:
Oh, wait. It's only the comments after operators, that's not so much. And I strongly prefer to move the operators to the beginning of the next line for multi-line expressions. So this is no problem, just the operators need to be moved.